api-client/src/testing/PoiModelBuilder.ts

```diff
@@ -25,8 +25,8 @@ const pois = [
       address: 'Test Address 1',
       town: 'Test Town',
       postcode: '12345',
-      latitude: 29.979848,
       longitude: 31.133859,
+      latitude: 29.979848,
       name: 'Test Title',
```

Comment:
pls reverted if change not needed

Reply:
I think it makes it easier to understand to have them in this order because that's the same order as in the coordinates.